sdk/dslink/src/main/java/org/dsa/iot/dslink/serializer/Deserializer.java

```diff
@@ -15,6 +15,10 @@
     private final SerializationManager serializationManager;
     private final NodeManager nodeManager;
 
+    public Deserializer(NodeManager nodeManager) {
+        this(nodeManager.getSuperRoot().getLink().getSerialManager(),nodeManager);
```

Comment:
bad formatting.
Also think you had the same formatting error with the last PR (missing space with the comma)